libsolidity/analysis/TypeChecker.cpp

```diff
@@ -3057,7 +3123,10 @@ void TypeChecker::requireLValue(Expression const& _expression, bool _ordinaryAss
 	if (_expression.annotation().isLValue)
 		return;
 
-	return m_errorReporter.typeError(_expression.location(), [&]() {
+	return m_errorReporter.typeError(
+		1938_uid,
+		_expression.location(),
+		[&]() {
 		if (_expression.annotation().isConstant)
 			return "Cannot assign to a constant variable.";
 
```

Comment:
There are cases where we'd likely need to refactor first. See for instance the last change, where a lambda is used to return different messages.